### ISSUE TITLE
Travis CI issue

### ISSUE THREAD

**Issue by wing328** [OpenAPITools/openapi-generator#117]
##### Description

Travis CI timed out due to the following:
```
openapi-petstore-0.1.0.0: Test suite tests passed
Completed 14 action(s).
Compiling src/Byte.elm
Some new packages are needed. Here is the upgrade plan.

  Install:
    NoRedInk/elm-decode-pipeline 3.0.0
    elm-lang/core 5.1.1
    elm-lang/html 2.0.0
    elm-lang/http 1.0.0
    elm-lang/virtual-dom 2.0.4
    justinmimbs/elm-date-extra 2.1.1
```
##### openapi-generator version

Current master

##### Related issues/PRs

https://travis-ci.org/OpenAPITools/openapi-generator/builds/380911655

##### Suggest a fix/enhancement

We may want do disable the elm client test for the time being to see if the issue goes away

**Comment by FantasyTeddy** [comment]
This issue should be resolved with PR #118 (f222245).

**Comment by wing328** [comment]
Yup, thanks for the PR to fix that.
